ckan/lib/munge.py

```diff
@@ -156,8 +158,8 @@ def munge_filename(filename):
     # Clean up
     filename = filename.lower().strip()
     filename = substitute_ascii_equivalents(filename)
-    filename = re.sub(ur'[^a-zA-Z0-9_. -]', '', filename).replace(u' ', u'-')
-    filename = re.sub(ur'-+', u'-', filename)
+    filename = re.sub(r'[^a-zA-Z0-9_. -]', '', filename).replace(u' ', u'-')
+    filename = re.sub(r'-+', u'-', filename)
```

Comment:
`r''` only matters for strings with backslashes, the better change is to make these `u''` strings.

Reply:
You did read through @torfsen notes at https://github.com/ckan/ckan/issues/4039#issuecomment-369227934 ?  I can go either way but you two need to be in agreement.